### PR TITLE
Increase Polygon service's observation interval

### PR DESCRIPTION
**Pull request by Adamantios**
## Proposed changes

Increase Polygon service's observation interval.

## Fixes

n/a

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [x] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

n/a

### CODE REVIEW

File: packages/valory/services/oracle_polygon/service.yaml [hunk 1]
```diff
@@ -10,14 +10,15 @@ fingerprint_ignore_patterns: []
 agent: valory/oracle:0.1.0:bafybeifperyoibdy3fitngdpr4ou6ig5t5xflje7g4dgcajuzwbymzxb64
 number_of_agents: 4
 ---
-setup_args:
+params_args:
```
Comment (dvilelaf) [inline packages/valory/services/oracle_polygon/service.yaml:13]: Why this change?

Comment (Adamantios) [reply packages/valory/services/oracle_polygon/service.yaml:13]: Ahh sorry, forgot to comment. These are not anymore only for setup arguments, but also for other parameters.

**Review by dvilelaf — APPROVED** [review]
LGTM

**Review by DavidMinarsch — APPROVED** [review]
